server/controllers/userController.js

```diff
@@ -1,22 +1,38 @@
+import bcrypt from 'bcrypt';
 import Model from '../models';
+import Authenticator from '../middlewares/authenticator';
 
 const { User } = Model;
```

Comment:
`import {User} from '../models';`

Reply:
I tried this solution it didn't work. I went to stack overflow and this is the solution https://github.com/andela/phlox-ah/pull/8